todo_app/data/view_model.py

```diff
@@ -0,0 +1,31 @@
+class ViewModel:
+
+    def __init__(self, items, list_targets):
+        self._items = items
+        self._list_targets = list_targets
+
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def list_targets(self):
+        return self._list_targets
+
+    @property
+    def to_do_items(self):
+        to_do_items_id = self.list_targets.get("To Do")
+        to_do_items = [item for item in self.items if item.idList == to_do_items_id]
```

Comment:
You can potentially pull this out into a separate function since you're repeating the same thing for all three lists just with a different id but its not necessary if you'd rather not to